services/dask-sidecar/src/simcore_service_dask_sidecar/file_utils.py

```diff
@@ -136,7 +136,7 @@ async def pull_file_from_remote(
     if not target_mime_type:
         target_mime_type, _ = mimetypes.guess_type(dst_path)
 
-    storage_kwargs = {}
+    storage_kwargs: Union[S3FsSettingsDict, dict[str, Any]] = {}
```

Comment:
Q: Does it work with ``S3FsSettingsDict`` alone? if not , why?

Reply:
Unfortunately not as S3FsSettingsDict is TypedDict, it complains:
```
error: Missing keys ("key", "secret", "token", "use_ssl", "client_kwargs") for TypedDict "S3FsSettingsDict"
```
I can leave it like this, or otherwise, modify S3FsSettingsDict variables to be Optional[...]